### PR TITLE
Backend MVP Complete

### PR DESCRIPTION
**Pull request by dishadc**

### CODE REVIEW

File: Backend/tests/test_flask.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+from flask import Flask
+
+import sys
+from app import app
+
+
+@pytest.fixture
+def client():
+    # db_fd, flaskr.app.config['DATABASE'] = tempfile.mkstemp()
```
Comment (yaretzj) [inline Backend/tests/test_flask.py]: Remove Commented code

Comment (dishadc) [reply Backend/tests/test_flask.py]: Comments removed

**Review by phiphphi — APPROVED** [review]
Looks good!

File: Backend/tests/test_flask.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+def client():
+    app.config["TESTING"] = True
+    with app.test_client() as client:
+        yield client
+
+
+def test_hello_world(client):
+    rv = client.get("/")
+    print(rv.data)
+    # print("hello, world")
```
Comment (yaretzj) [inline Backend/tests/test_flask.py]: Remove comment

File: Backend/db/queries.py [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+def update_chore(columns: list) -> str:
+    """Update the status for a chore (given id) to the given status."""
+    query = Query.update(chores_table).where(chores_table.ChoreId == Parameter("?"))
+    for column in columns:
+        query = query.set(Field(column), Parameter("?"))
+    return query.get_sql()
+
+
+# Test to show example generated SQL string
+# print(get_child_by_account_id(("Name", "Points")))
```
Comment (yaretzj) [inline Backend/db/queries.py]: Remove comments

File: Backend/app.py [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+
+    chore = cursor.fetchone()
+    if not chore or chore.ParentGoogleAccountId != child_acc.ParentGoogleAccountId:
+        abort(404, "ChoreId {} does not exist".format(chore_id))
+
+    update_values = [body[key] for key in columns]
+
+    try:
+        db_conn.autocommit = False
+        print(queries.update_chore(columns))
```
Comment (yaretzj) [inline Backend/app.py]: Comment out or delete this print

**Pull request merged**